### ISSUE TITLE
[BUG] Update process / Update popup missing

### ISSUE THREAD

**Issue by meeple** [aonez/Keka#1540]
### Configuration
- Keka version: 1.4.4 (german language version, download from webpage)
- macOS version: 14.7.1

### Describe the bug
Search for Update does not work, if there is an update.

### To Reproduce
1. Go to Menue
2. Click on search for update
3. the Update-popup is missing

### Expected behavior
If my keka version is the latest, there will be a popup to notify, that everything is fine.
If my keka version is not the latest, there simply isn't any notification or popup. I think this was also present in prior versions, but it did work at some point in the past.

**Comment by gingerbeardman** [comment]
Just some notes:

if the app is running under translocation (macOS security measure) you would see this:

solve that by moving the app out of /Applications and back in.

<img width="372" alt="Screen shot 2024-11-02 at 14 11 27" src="https://github.com/user-attachments/assets/930b1880-8a2b-452f-978b-3702c8e0d9ae">

checking for updates should show a progress:

<img width="512" alt="Screen shot 2024-11-02 at 14 13 05" src="https://github.com/user-attachments/assets/234899b0-95e5-40e2-989f-4f790ec06d80">

and then the update changelog box:

<img width="732" alt="Screen shot 2024-11-02 at 14 13 08" src="https://github.com/user-attachments/assets/4a5968bf-df56-4bdf-b0b1-daca1c93b0a5">

**Comment by aonez** [comment]
@meeple checked and works properly for me (and @gingerbeardman). Please execute this on the Terminal and share the strings you're seeing there:

```sh
defaults read com.aone.keka  | grep '^    SU'
```

You should see something similar to this:

```sh
    SUAutomaticallyUpdate = 1;
    SULastCheckTime = "2024-11-02 14:21:30 +0000";
```

**Comment by meeple** [comment]
I did get a few more lines in terminal:

>   SUAutomaticallyUpdate = 1;
>   SUEnableAutomaticChecks = 1;
>   SUHasLaunchedBefore = 1;
>   SULastCheckTime = "2024-11-02 13:00:14 +0000";
>   SUUpdateGroupIdentifier = 3788080339;

I never got the update error popup. The "checking for updates" was there and followed by
![popup](https://github.com/user-attachments/assets/f1d1836c-895b-48fa-a431-472adcc76de8)
This was always there until the moment a new version was ready. From this moment on, there simply was no popup at all.

However, I already updated to 1.4.5 by manually downloading and installing. Today I went back to 1.4.4 and now I get the update popup. It's weird, but I guess, it is solved for now.

Thank you and keep up the great work.

**Comment by aonez** [comment]
@meeple you have "Automatically install new updates" checked (`SUEnableAutomaticChecks`) so it should keep updated automatically for you. Disable that option if you prefer to see the update pop up:
<img width="801" alt="Screenshot 2024-11-02 at 21 36 17" src="https://github.com/user-attachments/assets/13666180-4eff-42d0-9478-f2c4a1072f4b">

You can always try with an older version like [v1.4.4](https://github.com/aonez/Keka/releases/tag/v1.4.4)

**Comment by meeple** [comment]
Ok. That means, the update is already in process when I do request "check for updates". I think it would be nice to have an info about that, because staying silent looked like a broken update routine to to me.

**Comment by aonez** [comment]
@meeple just released version v1.4.6. You can remove the last check date to force a check (previous to open Keka):

```sh
defaults remove com.aone.keka SULastCheckTime
```
